csharp/src/Ice/ITransceiver.cs

```diff
@@ -15,6 +15,12 @@ namespace ZeroC.Ice
 
     public interface ITransceiver
     {
+        public Connection CreateConnection(
```

Comment:
I think it would be more logical to:
  - make `Endpoint` the first parameter
  - give Endpoint a Communicator get-only public property
  - remove the Communicator parameter

Also, a doc-comment for ITransceiver would be great!  Can multiple endpoints use the same transceiver?

Reply:
Fixed
> Can multiple endpoints use the same transceiver?

For incoming connections there are multiple connections per endpoint, for outgoing connections AFAIK we always reuse one if it is available.